roles/splunk_common/tasks/check_mgmt_mode_status.yml

```diff
@@ -7,14 +7,19 @@
 - name: Check if listening on SVC Port {{ splunk.svc_port }}
   shell: "netstat -lnt"
   register: port_status
-  #when: not client_socket_file.stat.exists
+  when: ansible_facts['os_family'] != 'Ubuntu'
+
+- name: (Ubuntu) Check if listening on SVC Port {{ splunk.svc_port }}
+  shell: "ss -lnt"
+  register: port_status_ubuntu
+  when: ansible_facts['os_family'] == 'Ubuntu'
 
 - name: UF is configured to use UDS
   debug:
     msg: "This UF instance is configured to use UDS socket for API communications."
-  when: client_socket_file.stat.exists and port_status.stdout.find("{{ splunk.svc_port }}") == -1
+  when: (client_socket_file.stat.exists and port_status.skipped is false and port_status.stdout.find("{{ splunk.svc_port }}") == -1) or (client_socket_file.stat.exists and port_status_ubuntu.skipped is false and port_status_ubuntu.stdout.find("{{ splunk.svc_port }}") == -1)
 
 - name: UF is configured to use TCP
   debug:
     msg: "This UF instance is configured to use TCP socket for API communications."
-  when: not client_socket_file.stat.exists and port_status.stdout.find("{{ splunk.svc_port }}") != -1
+  when: (not client_socket_file.stat.exists and port_status.skipped is false and port_status.stdout.find("{{ splunk.svc_port }}") != -1) or (not client_socket_file.stat.exists and port_status_ubuntu.skipped is false and port_status_ubuntu.stdout.find("{{ splunk.svc_port }}") != -1)
```

Comment:
We can probably just infer if UF is configured to use UDS or TCP by seeing if Splunk has produced the socket file. We just keep our business simple and don't actually go check if any HTTP port has been opened. We just assume that is the case because otherwise splunk-ansible wouldn't have been able to complete provisioning